content/install_maintain/installation/installing-cluster.md

```diff
@@ -44,69 +42,14 @@ In case you use an Externally hosted PostgreSQL or RabbitMQ, i.e. "bring your ow
 read the relevant information for this case. 
 {{% /note %}}  
 
-{{% note title="VMs setup" %}}  
-Before you proceed, make sure you have all VMs spinning, that they are all allocated with a public-ip, and that they are configured according
+**Note:** Before you proceed, make sure you have al l VMs spinning, that they are all allocated with a public-ip, and that they are configured according
 to the [prerequisites guide] ({{< relref "install_maintain/installation/prerequisites.md" >}}).
 If you use Cloudify best-practice, you would need 10 VMs spinning: 3 PostgreSQL nodes, 3 RabbitMQ nodes, 3 Cloudify Management service nodes, 
```

Comment:
9 VMS + a load balancer

Reply:
Changed to: If you use Cloudify best-practice, you would need 9 VMs + a load balancer. The VMs partitioning is 3 PostgreSQL nodes, 3 RabbitMQ nodes, and 3 Cloudify Management service nodes.